app/javascript/packages/document-capture/components/address-search.tsx

```diff
@@ -184,9 +185,9 @@ function AddressSearch({
   }, [isLoading]);
 
   useEffect(() => {
-    locationResults && onFoundLocations(locationResults);
-    foundAddress && onFoundAddress(foundAddress);
-  }, [locationResults, foundAddress]);
+    !isLoading && locationResults && onFoundLocations(locationResults);
+    !isLoading && onFoundAddress(foundAddress);
+  }, [isLoading, locationResults, foundAddress]);
```

Comment:
I'll look into it!  I liked my approach because it caused the full result of # of locations near an address and the address to render at the same time.  I can try this approach, too. 👍🏻

Reply:
I made this change and am resolving the conversation. Thanks again!